src/components/shared/Types.tsx

```diff
@@ -5,7 +5,7 @@ import {
   SchemaCombinerName,
   SchemaNodeKind,
 } from '@stoplight/json-schema-tree';
-import cn from 'classnames';
+import { Box, Text } from '@stoplight/mosaic';
 import * as React from 'react';
 
 import { PROPERTY_TYPE_COLORS } from '../../consts';
```

Comment:
With your changes, you could eliminate the entire `PROPERTY_TYPE_COLORS` structure.

The only usage is in this file, but we can replace `keyof typeof PTC` with `SchemaNodeKind | SchemaCombinerName | '$ref'`, or hell, even `string` (the method handles any string just fine), and you're done. -20 lines 🚀